browser/extensions/api/brave_rewards_api.cc

```diff
@@ -270,5 +270,25 @@ void BraveRewardsGetACEnabledFunction::OnGetACEnabled(bool enabled) {
   Respond(OneArgument(std::make_unique<base::Value>(enabled)));
 }
 
+BraveRewardsSaveSettingFunction::~BraveRewardsSaveSettingFunction() {
+}
+
+ExtensionFunction::ResponseAction BraveRewardsSaveSettingFunction::Run() {
```

Comment:
I kept this as a mimic of UI's `saveSetting` rather than a standalone function for enabling rewards. This way if we add more configuration for rewards settings via the panel we can just add checks to this method rather than creating more separate ones.

Reply:
I like it